mmclient/http.go

```diff
@@ -12,8 +13,8 @@ type httpClient struct {
 	mm *model.Client4
 }
 
-func NewHTTPClient(mm *pluginapi.Client, conf config.Config, sessionID, actingUserID string) (Client, error) {
-	client, err := utils.ClientFromSession(mm, conf.MattermostSiteURL, sessionID, actingUserID)
+func NewHTTPClient(mm *pluginapi.Client, conf config.Config, sessionID, actingUserID string, i18nBundle *i18n.Bundle) (Client, error) {
```

Comment:
1/5 we should consolidate `mm`, `conf`, `i18n`, and soon `log` into a struct, and pass it around/inherit in the services (it'd also implement a `Configure()`

Separate PR.

Reply:
@levb Do we need tickets for this and your concern above?